firmware/boot/config.h

```diff
@@ -61,6 +61,15 @@
 #define BOOT_GPIO_PIN GPIO_PIN_5
 #define BOOT_FDCAN1
 
+#elif defined(quadruna_BMS)
+
+#define START_UPDATE_ID 1210
+#define UPDATE_ACK_ID 1211
+#define STATUS_10HZ_ID 1219
+#define BOOT_GPIO_PORT GPIOB
+#define BOOT_GPIO_PIN GPIO_PIN_5
+#define BOOT_FDCAN1
+
```

Comment:
These CAN IDs need to be different than the VCs, this is how each bootloader is uniquely identified on the bus. 
The convention I've been running with is 11xx IDs are for Thruna, 12xx IDs are for Quadruna, and 13xx IDs are for dev boards. And then a range of 10 is reserved for each board in the correct 100 range. Also fixed the pin by looking at Altium schematics.

```
#define START_UPDATE_ID 1200
#define UPDATE_ACK_ID 1201
#define STATUS_10HZ_ID 1209
#define BOOT_GPIO_PORT GPIOC
#define BOOT_GPIO_PIN GPIO_PIN_13
```

Reply:
Can you also add this to in `can_bus/quadruna/Boot` so this shows up in the DBC? (reference the VC for how to do it)

Reply:
is it boot_fdcan 1 or 2?

Reply:
FDCAN1
Whats the error?

Reply:
https://github.com/UBCFormulaElectric/Consolidated-Firmware/blob/e011f4572e47f1699bf69ea79c56c96212fe5d86/firmware/boot/h7boot/cubemx/Src/main.c#L154